lib/sqlalchemy_ingres/base.py

```diff
@@ -505,7 +534,10 @@ def get_columns(self, connection, table_name, schema=None, **kw):
                     coldata['type'] = ischema_names[coltype](precision, scale)
                 else:
                     coldata['type'] = ischema_names[coltype]
-                
+                coldata['autoincrement'] = (row[6] == 'Y' or row[7] == 'Y')
```

Comment:
Not required, a comment here on what 6/7 represent could be helpful here.

Reply:
Thanks for the review. Added comment for what row[6] and row[7] represent.